### PR TITLE
Fixed #35547 -- Added reset_queries to django.db.__all__.

### PR DESCRIPTION
**Pull request by AjmalPonneth**
# Trac ticket number

ticket-35547

# Branch description
This PR allows to import `reset_queries` from `django.db`.

# Checklist
- [x] This PR targets the `main` branch. 
- [ ] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [ ] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: docs/releases/5.0.5.txt [hunk 1]
```diff
@@ -33,3 +33,5 @@ Bugfixes
 
 * Fixed a bug in Django 5.0 where the ``querysets`` argument of
   ``GenericPrefetch`` was not required (:ticket:`35426`).
+
+* Allowed importing ``reset_queries`` from ``django.db`` (:ticket:`35547`).
```
Comment (sarahboyce) [inline docs/releases/5.0.5.txt:37]: We won't backport this into 5.0.5, can you please revert this change? ⭐

Comment (AjmalPonneth) [reply docs/releases/5.0.5.txt]: Done

**Review by sarahboyce — APPROVED** [review]
Thank you @AjmalPonneth - welcome onboard ⛵
